bin/release_automation.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# Check that Github CLI is installed
+command -v gh >/dev/null || { echo "Error: The Github CLI must be installed."; exit 1; }
+
+# Execute script commands from project's root directory
+SCRIPT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPT_PATH/.."
+
+# Check current branch is develop, master, or release/* branch
+CURRENT_BRANCH=$(git rev-parse --abbrev-ref HEAD)
+if [[ ! "$CURRENT_BRANCH" =~ "^develop$|^master$|^release/.*" ]]; then
+    echo "Releases should generally only be based on 'develop', 'master', or an earlier release branch."
+    echo "You are currently on the '$CURRENT_BRANCH' branch."
+    read -p "Are you sure you want to create a release branch from the '$CURRENT_BRANCH' branch? (y/n) " -n 1
+    echo ""
+    if [[ ! $REPLY =~ ^[Yy]$ ]]; then
+        printf "Aborting release...\n"
+        exit 1
+    fi
+fi
+
+# Confirm branch is clean
+[ -z "$(git status --porcelain)" ] || { git status; printf "\nUncommitted changes found. Aborting release script...\n"; exit 1; }
+
+# Ask for new version number
+CURRENT_VERSION_NUMBER=$(./node_modules/.bin/json -f package.json version)
+echo "Current Version Number:$CURRENT_VERSION_NUMBER"
+read -p "Enter the new version number: " VERSION_NUMBER
+
+# Insure javascript dependencies are up-to-date
+yarn install || { echo "Error: 'yarn install' failed"; echo 1; }
+
+# Create Git branch
+RELEASE_BRANCH="release/$VERSION_NUMBER"
+git switch -c "$RELEASE_BRANCH" || { echo "Error: could not create '$RELEASE_BRANCH' branch."; exit 1; }
+
+# Set version number in package.json
+yarn json -I -f package.json -e "this.version='$VERSION_NUMBER'" || { echo "Error: could not update version in package.json"; exit 1; }
+
+# Update the bundles
+yarn bundle || { echo "Error: 'yarn bundle' failed"; exit 1; }
+
+# Make sure podfile is updated
+pod install --project-directory=./ios/ || { echo "Error: pod install failed"; exit 1; }
```

Comment:
Had some issues with this command even after updating the path: p1592938673087400-slack-mobile-gutenberg-monorepo

Also, this now makes the changes in the `gutenberg/`repo, so we need to update the script to insure that gets committed too. This will essentially force us to move forward with having this script also handling the gutenberg side of the release like you suggested.

Reply:
I'm not seeing this issue anymore. 👍